extensions/wikia/PortabilityDashboard/PortabilityDashboard.setup.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+$dir = dirname( __FILE__ ) . '/';
+
+$wgAutoloadClasses[ 'PortabilityDashboardModel' ] = $dir . 'PortabilityDashboardModel.class.php';
+$wgAutoloadClasses[ 'PortabilityDashboardHooks' ] = $dir . 'PortabilityDashboardHooks.class.php';
+$wgAutoloadClasses[ 'SpecialPortabilityDashboardController' ] = $dir . 'SpecialPortabilityDashboardController.php';
+
+$wgSpecialPages[ 'PortabilityDashboard' ] = 'SpecialPortabilityDashboardController';
+$wgSpecialPageGroups[ 'PortabilityDashboard' ] = 'wikia';
+
+$wgHooks[ 'UnconvertedInfoboxesQueryRecached' ][] = 'PortabilityDashboardHooks::updateUnconvertedInfoboxes';
```

Comment:
According to what I see I think that if we hook like this, the hooks will get triggered only on wikis with PortabilityDashboard enabled; Is that intended?